### DISCUSSION TITLE
Would it happen when I only have 2 cluster nodes?

### DISCUSSION THREAD

**Discussion by sincorchetes** [longhorn/longhorn#8117] [General]
Problems?

**Comment by derekbit** [comment]
Longhorn doesn't rely on quorum, so 2-nodes cluster is fine.

**Reply by sincorchetes** [reply]
It's okay, but according to the minimum requirements, it needs at least 3 nodes to function properly, as outlined in the Longhorn documentation (https://longhorn.io/docs/1.6.0/best-practices/#architecture).

For example, I have read some posts advising against using this system with fewer than 3 nodes:

https://www.civo.com/marketplace/longhorn
https://www.suse.com/es-es/suse-longhorn/support-matrix/all-supported-versions/longhorn-v1-4-x/
Which components would not function correctly with only 2 nodes?
